recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ requirements:
     - setuptools
   run:
     - python
-    - numpy
+    - numpy x.x
```

Comment:
No problem. That is quite confusing. Please let me know if you have any comments on how to improve the docs to help clarify the use of `numpy x.x`
(We should also point to ways to identify when that is needed...)